.github/workflows/build-validation.yml

```diff
@@ -42,7 +41,7 @@ jobs:
       if: ${{ env.DOTNET_DO_INSTALL == 'true' }}
       run: |
         echo "Downloading dotnet-install.ps1"
-        Invoke-WebRequest https://raw.githubusercontent.com/dotnet/install-scripts/master/src/dotnet-install.ps1 -OutFile dotnet-install.ps1
+        Invoke-WebRequest https://raw.githubusercontent.com/dotnet/install-scripts/main/src/dotnet-install.ps1 -OutFile dotnet-install.ps1
```

Comment:
This repo still has master - https://github.com/dotnet/install-scripts

Reply:
Thanks for catching that! I thought it was pointing to this repo for some reason. They probably can't ever change since they promote the use of their scripts via deep linking.

Reply:
```suggestion
        Invoke-WebRequest https://raw.githubusercontent.com/dotnet/install-scripts/master/src/dotnet-install.ps1 -OutFile dotnet-install.ps1
```